source/locales.tex

```diff
@@ -1947,54 +1948,55 @@
 \indexlibrary{\idxcode{ctype<char>}}%
 \begin{codeblock}
 namespace std {
-  template <> class ctype<char>
-    : public locale::facet, public ctype_base {
-  public:
-    using char_type = char;
+  template <>
+    class ctype<char>
+      : public locale::facet, public ctype_base {
```

Comment:
Bring the base class up on the same line like everywhere else while you're at it.